### DISCUSSION TITLE
阿里云服务器场景下：docker部署时 net=host ，app_info.json读取的是无效的宿主机ip

### DISCUSSION THREAD

**Discussion by tushengtao** [alibaba/ilogtail#967] [Help]
（1）主要情况：具体复现情况，大家可查看此issue：https://github.com/alibaba/ilogtail/issues/957 ；目前我们解决方案是docker部署时传add-host参数来解决；当然也可以手动修改宿主机hosts文件，我们目前100+服务器，所以效率比较低，不可能手动修改hosts，所以传add-hosts的动态参数比较方便。
（2）ilogtail本身代码问题：当然最好的是尽可能完善下组件源码，不能直接默认取主机名在/etc/hosts文件里对应的第一个ip，没有进行有效ip的校验，关于前面说的代码问题，iLogTail组件的源码大家也可以去看看。
（3）自己的想法：目前自己还在思考比较轻量级的解决方法

**Comment by henryzhx8** [comment]
@TuShengTao hello，我们这周会举办最新一期的iLogtail双周会，您这个议题社区也有不少人提过，不知您是否方便在会上分享一下？方便的话能否告知一下您的钉钉号，我们在钉钉上细聊

**Reply by tushengtao** [reply]
很高兴收到你们的回复，你们的SLS产品在我们生产环境对日志收集和分析提供了很大的便利，这周六、周日我是方便的，我的钉钉号是：16z-ibkfd55nw8

**Reply by tushengtao** [reply]
你们都是选择周三晚上进行双周会嘛

**Reply by messixukejia** [reply]
一般都是周三晚上的。你这边有冲突吗？

**Reply by tushengtao** [reply]
目前看是没有的
